pycobertura/templates/html.jinja2

```diff
@@ -44,11 +51,15 @@ pre {line-height: 1.3}
           </tr>
         </tfoot>
       </table>
+{% if sources %}
 {%- from 'macro.source.jinja2' import render_source -%}
 {%- for filename, source in sources %}
 <h4 id="{{ filename }}">{{ filename }}</h4>
 {{ render_source(source) }}
 {%- endfor %}
+{% else %}
+    {{ sources_message }}
```

Comment:
Should this be wrapped in a `<p>` tag or something? Also, can we make the indentation in HTML files 2-space?

```suggestion
  <p>{{ sources_message }}</p>
```